Microsoft.VisualStudio.Terminal/TermWindowPackage.cs

```diff
@@ -37,6 +37,7 @@ namespace Microsoft.VisualStudio.Terminal
     [ProvideMenuResource("Menus.ctmenu", 1)]
     [ProvideToolWindow(typeof(TermWindow), Style = VsDockStyle.Tabbed, Window = "34E76E81-EE4A-11D0-AE2E-00A0C90FFFC3")]
     [ProvideToolWindow(typeof(ServiceToolWindow), Transient = true, MultiInstances = true, Style = VsDockStyle.Tabbed, Window = TermWindow.TermWindowGuidString)]
```

Comment:
Is there a case where we still need the old tool window? Couldn't it be implemented as a RendererToolWindow with a specific backend?

Reply:
I guess it can be implemented this way. I'll see if I can do that.